crates/polars-arrow/src/array/utf8.rs

```diff
@@ -55,13 +55,13 @@ impl<T: AsRef<str>> AsRef<[u8]> for StrAsBytes<T> {
 
 pub trait Utf8FromIter {
     #[inline]
-    fn from_values_iter<I, S>(iter: I, len: usize, value_cap: usize) -> Utf8Array<i64>
+    fn from_values_iter<I, S>(iter: I, len: usize, size_hint: usize) -> Utf8Array<i64>
```

Comment:
I think we must also add that it is the size_hint of the values buffer. So `size_hint_values`.